# Overview

Ph.D. Candidate at Vanderbilt University interested in high-dimensional data modeling and integration towards the characterization of colorectal cancer initiation, progression, and microenvironmental co-evolution.

#### Links

* [ORCID: 0000-0002-1299-1628](https://orcid.org/0000-0002-1299-1628)
* [Lau Lab Website](https://www.mc.vanderbilt.edu/vumcdept/cellbio/laulab/people.html)
* [Lau Lab Software](https://Ken-Lau-Lab.github.io)

# Education

#### Vanderbilt University

> Ph.D., [Chemical & Physical Biology](https://medschool.vanderbilt.edu/cpb/) 2022 (*expected*)  
> Thesis research in the [Lau Lab](https://Ken-Lau-Lab.github.io)

#### University of Texas at Austin

> B.S., [Biomedical Engineering](https://www.bme.utexas.edu/), 2016

# Publications

Chen B, Ramirez-Solano MA, **Heiser CN**, Liu Q, Lau KS. 2021. Processing single-cell RNA-seq data for dimension reduction-based analyses using open-source tools. *STAR Protocols*. Cell Press. [doi: 10.1016/j.xpro.2021.100450](https://doi.org/10.1016/j.xpro.2021.100450)

**Heiser CN**, Wang VM, Chen B, Hughey JJ, Lau KS. 2021. Automated quality control and cell identification of droplet-based single-cell data using dropkick. *Genome Research*. Cold Spring Harbor Press. [doi: 10.1101/gr.271908.120](https://doi.org/10.1101/gr.271908.120)

Lin J-R, Wang S, Coy S, Tyler MA, Yapp C, Chen Y-A, **Heiser CN**, Lau KS, Santagata S, Sorger PK. 2021. Multiplexed 3D atlas of state transitions and immune interactions in colorectal cancer. *bioRxiv*. Cold Spring Harbor Press. [doi: 10.1101/2021.03.31.437984](http://doi.org/10.1101/2021.03.31.437984).

Bao S, Chiron S, Tang Y, **Heiser CN**, et al. A cross-platform informatics system for the Gut Cell Atlas: integrating clinical, anatomical and histological data. *Proc. SPIE 11601, Medical Imaging 2021: Imaging Informatics for Healthcare, Research, and Applications*, 1160106. 15 Feb 2021. [doi: 10.1117/12.2581074](https://doi.org/10.1117/12.2581074).

Chen B, McKinley ET, Simmons AJ, Ramirez-Solano MA, Zhu X, Southard-Smith AN, Markham NO, Sheng Q, Drewes JL, Xu Y, **Heiser CN**, et al. 2021. Human colorectal pre-cancer atlas identifies distinct molecular programs underlying two major subclasses of pre-malignant tumors. *bioRxiv*. Cold Spring Harbor Press. [doi: 10.1101/2021.01.11.426044](https://doi.org/10.1101/2021.01.11.426044).

**Heiser CN** and Lau KS. 2020. A quantitative framework for evaluating single-cell data structure preservation by dimensionality reduction techniques. *Cell Reports*. Cell Press, 31(5), p. 107576. [doi: 10.1016/j.celrep.2020.107576](https://doi.org/10.1016/j.celrep.2020.107576).

Rozenblatt-Rosen O, et al. 2020. The Human Tumor Atlas Network: charting tumor transitions across space and time at single-cell resolution, *Cell*. Cell Press, 181(2), pp. 236–249. [doi: 10.1016/J.CELL.2020.03.053](https://doi.org/10.1016/J.CELL.2020.03.053).

Yang X, Graff SM, **Heiser CN**, et al. 2020. Coregulator Sin3a promotes postnatal murine β-cell fitness by regulating genes in Ca 2+ homeostasis, cell survival, vesicle biosynthesis, glucose metabolism, and stress response. *Diabetes*. p. db190721. [doi: 10.2337/db19-0721](https://doi.org/10.2337/db19-0721).

# Code

[`dropkick`](https://github.com/KenLauLab/dropkick): automated QC and cell identification of <acronym title="single-cell RNA sequencing">scRNA-seq</acronym> data ([documentation](https://kenlaulab.github.io/dropkick/))

[![Latest Version][pypi-image-dropkick]][pypi-url-dropkick]

[`STAR_protocol`](https://github.com/KenLauLab/STAR_Protocol): standard processing workflow for <acronym title="single-cell RNA sequencing">scRNA-seq</acronym> data

[`kitchen`](https://github.com/codyheiser/kitchen): automated manipulation and processing of <acronym title="single-cell RNA sequencing">scRNA-seq</acronym> counts files from command line ([documentation](https://codyheiser.github.io/kitchen/))

[![Latest Version][tag-version-kitchen]][repo-url-kitchen]

[`cnmf`](https://github.com/codyheiser/cnmf): packaged and streamlined version of <acronym title="consensus nonnegative matrix factorization">cNMF</acronym>, adapted from [Kotliar, et al. 2019](https://github.com/dylkot/cNMF) ([documentation](https://codyheiser.github.io/cnmf/))

[![Latest Version][tag-version-cnmf]][repo-url-cnmf]

[`DR-structure`](https://github.com/KenLauLab/DR-structure-preservation): quantitative framework for evaluating structural preservation of dimension reduced embeddings ([documentation](https://kenlaulab.github.io/DR-structure-preservation/))

[pypi-image-dropkick]: https://img.shields.io/pypi/v/dropkick
[pypi-url-dropkick]: https://pypi.python.org/pypi/dropkick/
[tag-version-kitchen]: https://img.shields.io/github/v/tag/codyheiser/kitchen
[repo-url-kitchen]: https://github.com/codyheiser/kitchen
[tag-version-cnmf]: https://img.shields.io/github/v/tag/codyheiser/cNMF
[repo-url-cnmf]: https://github.com/codyheiser/cNMF
