# Bio

Ph.D. computational systems biologist interested in high-dimensional data modeling and integration towards the characterization of cancer initiation, progression, and microenvironmental co-evolution.

#### Links

* [ResearchGate](https://www.researchgate.net/profile/Cody-Heiser)
* [ORCID: 0000-0002-1299-1628](https://orcid.org/0000-0002-1299-1628)
* [Lau Lab Website](https://kenlaulab.org/)
* [Lau Lab Software](https://Ken-Lau-Lab.github.io)

# Education

#### Vanderbilt University

> Ph.D., [Chemical & Physical Biology](https://medschool.vanderbilt.edu/cpb/), 2023<br />Dissertation: ["Molecular cartography uncovers evolutionary and microenvironmental dynamics in sporadic colorectal tumors"](http://hdl.handle.net/1803/18183)

#### University of Texas at Austin

> B.S., [Biomedical Engineering](https://www.bme.utexas.edu/), 2016

# Selected Publications

**Heiser CN**, Simmons AJ, Revetta F, McKinley ET, Ramirez-Solano MA, Wang J, Shao J, Ayers GD, Wang Y, Glass SE, Kaur H, Rolong A, Chen B, Vega PN, Drewes JL, Saleh N, Vandekar S, Washington MK, Roland JT, Sears CL, Liu Q, Shrubsole MJ, Coffey RJ, Lau KS. 2023. Molecular cartography uncovers evolutionary and microenvironmental dynamics in sporadic colorectal tumors. In revision at *Cell*. [Preprint doi: 10.1101/2023.03.09.530832](https://doi.org/10.1101/2023.03.09.530832).

Kaur H\*, **Heiser CN**\*, McKinley ET, Ventura-Antunes L, Harris CR, Roland JT, Shrubsole MJ, Coffey RJ, Lau KS, Vandekar S. 2023. Consensus tissue domain detection in spatial multi-omics data using MILWRM. In revision at *Cell Systems*. [Preprint doi: 10.1101/2023.02.02.526900](https://doi.org/10.1101/2023.02.02.526900). <br />
[![Latest Version][pypi-image-MILWRM]][pypi-url-MILWRM] [`MILWRM` package](https://github.com/Ken-Lau-Lab/MILWRM) & [documentation](https://ken-lau-lab.github.io/MILWRM/)

Arceneaux D, Chen Z, Simmons AJ, **Heiser CN**, Southard-Smith AN, Brenan MJ, Yang Y, Chen B, Xu Y, Choi E, Campbell JD, Liu Q, Lau KS. 2023. A contamination focused approach for optimizing the single-cell RNA-seq experiment. In press at *iScience*. [Preprint doi: 10.1101/2022.10.25.513758](https://doi.org/10.1101/2022.10.25.513758).

Lin J, Wang S, Coy S, Chen Y, Yapp C, Tyler M, Nariya MK, **Heiser CN**, Lau KS, Santagata S, Sorger PK. 2023. Multiplexed 3D atlas of state transitions and immune interaction in colorectal cancer. *Cell*. Cell Press. [doi: 10.1016/j.cell.2022.12.028](https://doi.org/10.1016/j.cell.2022.12.028).

Drewes JL, Chen J, Markham NO, Knippel RJ, Domingue JC, Tam AJ, Chan JL, Kim L, McMann M, Stevens C, Dejea CM, Tomkovich S, Michel J, White JR, Mohammad F, Campodonico VL, **Heiser CN**, et al. 2022. Human colon cancer-derived *Clostridioides difficile* strains drive colonic tumorigenesis in mice. *Cancer Discovery*. American Association for Cancer Research. [doi: 10.1158/2159-8290.CD-21-1273](https://doi.org/10.1158/2159-8290.CD-21-1273).

McKinley ET, Shao J, Ellis ST, **Heiser CN**, Roland JT, Macedonia MC, Vega PN, Shin S, Coffey RJ, Lau KS. 2022. MIRIAM: A machine and deep learning single-cell segmentation and quantification pipeline for multi-dimensional tissue images. *Cytometry: Part A*. Wiley. [doi: 10.1002/cyto.a.24541](https://doi.org/10.1002/cyto.a.24541).

Chen B, Scurrah CR, McKinley ET, Simmons AJ, Ramirez-Solano MA, Zhu X, Markham NO, **Heiser CN**, et al. 2021 Differential pre-malignant programs and microenvironment chart distinct paths to malignancy in human colorectal polyps. *Cell*. Cell Press. [doi: 10.1016/j.cell.2021.11.031](https://doi.org/10.1016/j.cell.2021.11.031).

Wylezinski LS, Harris CR, **Heiser CN**, Gray JD and Spurlock CF. 2021. Influence of social determinants of health and county vaccination rates on machine learning models to predict COVID-19 case growth in Tennessee. *BMJ Health & Care Informatics*. BMJ, 0:e100439. [doi: 10.1136/bmjhci-2021-100439](http://doi.org/10.1136/bmjhci-2021-100439).

Chen B, Ramirez-Solano MA, **Heiser CN**, Liu Q, Lau KS. 2021. Processing single-cell RNA-seq data for dimension reduction-based analyses using open-source tools. *STAR Protocols*. Cell Press. [doi: 10.1016/j.xpro.2021.100450](https://doi.org/10.1016/j.xpro.2021.100450). <br />
[`STAR_protocol` repository](https://github.com/KenLauLab/STAR_Protocol)

**Heiser CN**, Wang VM, Chen B, Hughey JJ, Lau KS. 2021. Automated quality control and cell identification of droplet-based single-cell data using dropkick. *Genome Research*. Cold Spring Harbor Press. [doi: 10.1101/gr.271908.120](https://doi.org/10.1101/gr.271908.120). <br />
[![Latest Version][pypi-image-dropkick]][pypi-url-dropkick] [`dropkick` package](https://github.com/KenLauLab/dropkick) & [documentation](https://kenlaulab.github.io/dropkick/)

Bao S, Chiron S, Tang Y, **Heiser CN**, et al. A cross-platform informatics system for the Gut Cell Atlas: integrating clinical, anatomical and histological data. *Proc. SPIE 11601, Medical Imaging 2021: Imaging Informatics for Healthcare, Research, and Applications*, 1160106. 15 Feb 2021. [doi: 10.1117/12.2581074](https://doi.org/10.1117/12.2581074).

Yang X, Graff SM, **Heiser CN**, et al. 2020. Coregulator Sin3a promotes postnatal murine Î²-cell fitness by regulating genes in Ca 2+ homeostasis, cell survival, vesicle biosynthesis, glucose metabolism, and stress response. *Diabetes*. p. db190721. [doi: 10.2337/db19-0721](https://doi.org/10.2337/db19-0721).

**Heiser CN** and Lau KS. 2020. A quantitative framework for evaluating single-cell data structure preservation by dimensionality reduction techniques. *Cell Reports*. Cell Press, 31(5), p. 107576. [doi: 10.1016/j.celrep.2020.107576](https://doi.org/10.1016/j.celrep.2020.107576). <br />
[`DR-structure` repository](https://github.com/KenLauLab/DR-structure-preservation) & [documentation](https://kenlaulab.github.io/DR-structure-preservation/)

# Additional Code

[`kitchen`](https://github.com/codyheiser/kitchen): automated manipulation and processing of <acronym title="single-cell RNA sequencing">scRNA-seq</acronym> counts files from command line ([documentation](https://codyheiser.github.io/kitchen/)) <br />
[![Latest Version][tag-version-kitchen]][repo-url-kitchen]

[`cnmf`](https://github.com/codyheiser/cnmf): packaged and streamlined version of <acronym title="consensus nonnegative matrix factorization">cNMF</acronym>, adapted from [Kotliar, et al. 2019](https://github.com/dylkot/cNMF) ([documentation](https://codyheiser.github.io/cnmf/)) <br />
[![Latest Version][tag-version-cnmf]][repo-url-cnmf]

<div class="github-widget" data-username="codyheiser"></div>
<script src="https://unpkg.com/github-card@1.2.1/dist/widget.js"></script>

# Awards

* Graduate Student of the Year **(Vanderbilt University Men's Ice Hockey - 04/2023)**
* Brighter Ventures Award **(Vanderbilt University School of Medicine - 06/2022)**
* Dean's Award for Exceptional Achievement **(Vanderbilt University School of Medicine - 07/2021)**
* Lai Sulin Endowed Cancer Research Scholarship **(Vanderbilt University - 04/2021)**
* Russell G. Hamilton Graduate Leadership Institute Travel Grant Award **(Vanderbilt University - 09/2019)**

[pypi-image-MILWRM]: https://img.shields.io/pypi/v/MILWRM
[pypi-url-MILWRM]: https://pypi.python.org/pypi/MILWRM/
[pypi-image-dropkick]: https://img.shields.io/pypi/v/dropkick
[pypi-url-dropkick]: https://pypi.python.org/pypi/dropkick/
[tag-version-kitchen]: https://img.shields.io/github/v/tag/codyheiser/kitchen
[repo-url-kitchen]: https://github.com/codyheiser/kitchen
[tag-version-cnmf]: https://img.shields.io/github/v/tag/codyheiser/cNMF
[repo-url-cnmf]: https://github.com/codyheiser/cNMF
