---
hide:
  - path
---

## Systems Biology

??? note "Molecular cartography uncovers evolutionary and microenvironmental dynamics in sporadic colorectal tumors<br />__Heiser, *et al.* (2023) - *Cell*__"
    __Contribution:__ Planned multimodal experiments using human FFPE biopsies at different stages of CRC development and molecular subtypes. Managed data generation - scRNA-seq, MxIF, LCM-WES. Analyzed all data from spatial atlas in conjunction with scRNA-seq CRC atlas (Chen, _et al._, 2021, _Cell_). Wrote manuscript.

    __Full Citation:__ **Heiser CN**, Simmons AJ, Revetta F, McKinley ET, Ramirez-Solano MA, Wang J, Kaur H, Shao J, Ayers GD, Wang Y, Glass SE, Tasneem N, Chen Z, Qin Y, Kim W, Rolong A, Chen B, Vega PN, Drewes JL, Markham NO, Saleh N, Nikolos F, Vandekar S, Jones AL, Washington MK, Roland JT, Chan KS, Schurpf T, Sears CL, Liu Q, Shrubsole MJ, Coffey RJ, Lau KS. 2023. Molecular cartography uncovers evolutionary and microenvironmental dynamics in sporadic colorectal tumors. *Cell*. Cell Press. [doi: 10.1016/j.cell.2023.11.006](https://doi.org/10.1016/j.cell.2023.11.006).

??? note "Multiplexed 3D atlas of state transitions and immune interaction in colorectal cancer<br />__Lin, *et al.* (2023) - *Cell*__"
    __Contribution:__ Analyzed single-cell RNA sequencing data to identify tumor cell clusters and marker genes using differential gene expression analysis.

    __Full Citation:__ Lin J, Wang S, Coy S, Chen Y, Yapp C, Tyler M, Nariya MK, **Heiser CN**, Lau KS, Santagata S, Sorger PK. 2023. Multiplexed 3D atlas of state transitions and immune interaction in colorectal cancer. *Cell*. Cell Press. [doi: 10.1016/j.cell.2022.12.028](https://doi.org/10.1016/j.cell.2022.12.028).

??? note "Differential pre-malignant programs and microenvironment chart distinct paths to malignancy in human colorectal polyps<br />__Chen, *et al.* (2021) - *Cell*__"
    __Contribution:__ Analyzed single-cell RNA sequencing data from atlas cohort to identify tumor and immune cell clusters and marker genes using differential gene expression analysis.

    __Full Citation:__ Chen B, Scurrah CR, McKinley ET, Simmons AJ, Ramirez-Solano MA, Zhu X, Markham NO, **Heiser CN**, Vega PN, Rolong A, Kim H, Sheng Q, Drewes JL, Zhou Y, Southard-Smith AN, Xu Y, Ro J, Jones AL, Revetta F, Berry LD, Niitsu H, Islam M, Pelka K, Hofree M, Chen JH, Sarkizova S, Ng K, Giannakis M, Boland GM, Aguirre AJ, Anderson AC, Rozenblatt-Rosen O, Regev A, Hacohen N, Kawasaki K, Sato T, Goettel JA, Grady WM, Zheng W, Washington MK, Cai Q, Sears CL, Goldenring JR, Franklin JL, Su T, Huh WJ, Vandekar S, Roland JT, Liu Q, Coffey RJ, Shrubsole MJ, Lau KS. 2021. Differential pre-malignant programs and microenvironment chart distinct paths to malignancy in human colorectal polyps. *Cell*. Cell Press. [doi: 10.1016/j.cell.2021.11.031](https://doi.org/10.1016/j.cell.2021.11.031).

## Oncology & Immunology

??? note "The differentiation and function of heterogeneous thymic dendritic cell subsets require signals provided by distinct thymocyte cell types<br />__Srinivasan, *et al.* (2026) - *Nature Immunology*__"
    __Contribution:__ Analyzed single-cell RNA sequencing data to identify dendritic cell clusters and marker genes using differential gene expression analysis.

    __Full Citation:__ Srinivasan J, Moore CR, Calindi A, Helm BR, Yang Y, Moore JF, Selden HJ, **Heiser CN**, Liu Q, Lau KS, Ehrlich LIR. 2026. The differentiation and function of heterogeneous thymic dendritic cell subsets require signals provided by distinct thymocyte cell types. *Nature Immunology*. [doi: 10.1038/s41590-025-02371-9](https://doi.org/10.1038/s41590-025-02371-9).

??? note "Dipeptidase-1 knockout mice develop invasive tumors with features of microsatellite unstable colorectal cancer<br />__Glass, *et al.* (2025) - *JCI Insight*__"
    __Contribution:__ Consulted data analysis using spatial CRC atlas data (Heiser, _et al._, 2023, _Cell_).

    __Full Citation:__ Glass SE, Bechard ME, Cao Z, Aramandla R, Zhao P, Ellis ST, Green EH, Fisher EG, Smith RT, Sievers CK, Johnson Irudayam M, Revetta F, Washington MK, Ayers GD, **Heiser CN**, Simmons AJ, Xu Y, Wang Y, Windon A, Shrubsole MJ, Markham NO, Liu Q, Lau KS, Coffey RJ. 2025. Dipeptidase-1 knockout mice develop invasive tumors with features of microsatellite unstable colorectal cancer. *JCI Insight*. [doi: 10.1172/jci.insight.186938](https://doi.org/10.1172/jci.insight.186938).

??? note "Multiomic spatial atlas shows deleted in malignant brain tumors 1 (DMBT1) glycoprotein is lost in colonic dysplasia<br />__Green, *et al.* (2025) - *Journal of Pathology*__"
    __Contribution:__ Consulted data analysis using spatial CRC atlas data (Heiser, _et al._, 2023, _Cell_).

    __Full Citation:__ Green EH, Kotrannavar SR, Rutherford ME, Lunnemann HM, Kaur H, **Heiser CN**, Ding H, Simmons AJ, Liu X, Lacy DB, Washington MK, Shrubsole MJ, Liu Q, Lau KS, Sears CL, Coffey RJ, Drewes JL, Markham NO. 2025. Multiomic spatial atlas shows deleted in malignant brain tumors 1 (DMBT1) glycoprotein is lost in colonic dysplasia. *Journal of Pathology*. [doi: 10.1002/path.6406](https://doi.org/10.1002/path.6406).

??? note "Human colon cancer-derived *Clostridioides difficile* strains drive colonic tumorigenesis in mice<br />__Drewes, *et al.* (2022) - *Cancer Discovery*__"
    __Contribution:__ Analyzed single-cell RNA sequencing data to identify clusters of tumor and tumor-infiltrating immune cells from mouse models of CRC innoculated with C. Diff. from human patients.

    __Full Citation:__ Drewes JL, Chen J, Markham NO, Knippel RJ, Domingue JC, Tam AJ, Chan JL, Kim L, McMann M, Stevens C, Dejea CM, Tomkovich S, Michel J, White JR, Mohammad F, Campodonico VL, **Heiser CN**, Wu X, Wu S, Ding H, Simner P, Carroll K, Shrubsole MJ, Anders RA, Walk ST, Jobin C, Wan F, Coffey RJ, Housseau F, Lau KS, Sears CL. 2022. Human colon cancer-derived *Clostridioides difficile* strains drive colonic tumorigenesis in mice. *Cancer Discovery*. American Association for Cancer Research. [doi: 10.1158/2159-8290.CD-21-1273](https://doi.org/10.1158/2159-8290.CD-21-1273).

## Computational Biology & Computer Science

??? note "Consensus tissue domain detection in spatial multi-omics data using multiplex image labeling with regional morphology (MILWRM)<br />__Kaur, *et al.* (2024) - *Communications Biology*__"
    __Contribution:__ Conceptualized unsupervised tissue region identification and clustering in multiplex immunofluorescence (MxIF) and spatial transcriptomics (ST) modalities. Wrote original Python package code, which was augmented and published by my colleague, Harsimran Kaur.

    __Full Citation:__ Kaur H, **Heiser CN**, McKinley ET, Ventura-Antunes L, Harris CR, Roland JT, Farrow MA, Selden HJ, Pingry EL, Moore JF, Ehrlich LIR, Shrubsole MJ, Spraggins JM, Coffey RJ, Lau KS, Vandekar SN. 2024. Consensus tissue domain detection in spatial multi-omics data using multiplex image labeling with regional morphology (MILWRM). *Communications Biology*. Springer Nature. [doi: 10.1038/s42003-024-06281-8](https://doi.org/10.1038/s42003-024-06281-8).

??? note "GammaGateR: semi-automated marker gating for single-cell multiplexed imaging<br />__Xiong, *et al.* (2024) - *Bioinformatics*__"
    __Contribution:__ Consulted on GammaGateR validation using multiplex immunofluorescence (MxIF) datasets.

    __Full Citation:__ Xiong J, Kaur H, **Heiser CN**, McKinley ET, Roland JT, Coffey RJ, Shrubsole MJ, Wrobel J, Ma S, Lau KS, Vandekar S. 2024. GammaGateR: semi-automated marker gating for single-cell multiplexed imaging. *Bioinformatics*. Oxford Academic. [doi: 10.1093/bioinformatics/btae356](http://doi.org/10.1093/bioinformatics/btae356).

??? note "A contamination focused approach for optimizing the single-cell RNA-seq experiment<br />__Arceneaux, *et al.* (2023) - *iScience*__"
    __Contribution:__ Added best-practices from `dropkick` QC and filtering to full end-to-end scRNA-seq analysis protocol.

    __Full Citation:__ Arceneaux D, Chen Z, Simmons AJ, **Heiser CN**, Southard-Smith AN, Brenan MJ, Yang Y, Chen B, Xu Y, Choi E, Campbell JD, Liu Q, Lau KS. 2023. A contamination focused approach for optimizing the single-cell RNA-seq experiment. *iScience*. Cell Press. [doi: 10.1016/j.isci.2023.107242](http://dx.doi.org/10.1016/j.isci.2023.107242).

??? note "MIRIAM: A machine and deep learning single-cell segmentation and quantification pipeline for multi-dimensional tissue images<br />__McKinley, *et al.* (2022) - *Cytometry: Part A*__"
    __Contribution:__ Tested and validated MIRIAM package on multiplex immunofluorescence (MxIF) dataset generated for my colorectal cancer atlas paper.

    __Full Citation:__ McKinley ET, Shao J, Ellis ST, **Heiser CN**, Roland JT, Macedonia MC, Vega PN, Shin S, Coffey RJ, Lau KS. 2022. MIRIAM: A machine and deep learning single-cell segmentation and quantification pipeline for multi-dimensional tissue images. *Cytometry: Part A*. Wiley. [doi: 10.1002/cyto.a.24541](https://doi.org/10.1002/cyto.a.24541).

??? note "Processing single-cell RNA-seq data for dimension reduction-based analyses using open-source tools<br />__Chen, *et al.* (2021) - *STAR Protocols*__"
    __Contribution:__ Added best-practices from `dropkick` QC and filtering to full end-to-end scRNA-seq analysis protocol.

    __Full Citation:__ Chen B, Ramirez-Solano MA, **Heiser CN**, Liu Q, Lau KS. 2021. Processing single-cell RNA-seq data for dimension reduction-based analyses using open-source tools. *STAR Protocols*. Cell Press. [doi: 10.1016/j.xpro.2021.100450](https://doi.org/10.1016/j.xpro.2021.100450).

??? note "Automated quality control and cell identification of droplet-based single-cell data using dropkick<br />__Heiser, *et al.* (2021) - *Genome Research*__"
    __Contribution:__ Conceptualized scRNA-seq droplet QC, ambient gene identification and quantification, and machine learning-based droplet filtering model. Wrote Python package compatible with `scanpy` scRNA-seq analysis ecosystem. Wrote manuscript.

    __Full Citation:__ **Heiser CN**, Wang VM, Chen B, Hughey JJ, Lau KS. 2021. Automated quality control and cell identification of droplet-based single-cell data using dropkick. *Genome Research*. Cold Spring Harbor Press. [doi: 10.1101/gr.271908.120](https://doi.org/10.1101/gr.271908.120).

??? note "A quantitative framework for evaluating single-cell data structure preservation by dimensionality reduction techniques<br />__Heiser & Lau (2020) - *Cell Reports*__"
    __Contribution:__ Conceptualized numerical methods for quantifying high-dimensional data structure. Wrote entire Python code base and paper.

    __Full Citation:__ **Heiser CN** and Lau KS. 2020. A quantitative framework for evaluating single-cell data structure preservation by dimensionality reduction techniques. *Cell Reports*. Cell Press, 31(5), p. 107576. [doi: 10.1016/j.celrep.2020.107576](https://doi.org/10.1016/j.celrep.2020.107576).

## Miscellaneous

??? note "Influence of social determinants of health and county vaccination rates on machine learning models to predict COVID-19 case growth in Tennessee<br />__Wylezinski, *et al.* (2021) - *BMJ Health & Care Informatics*__"
    __Contribution:__ Processed SDoH and COVID-19 outcomes data through automated machine learning pipeline, extracting weights from top models to identify explained drivers of COVID-19 cases by Tennessee county.
    
    __Full Citation:__ Wylezinski LS, Harris CR, **Heiser CN**, Gray JD and Spurlock CF. 2021. Influence of social determinants of health and county vaccination rates on machine learning models to predict COVID-19 case growth in Tennessee. *BMJ Health & Care Informatics*. BMJ, 0:e100439. [doi: 10.1136/bmjhci-2021-100439](http://doi.org/10.1136/bmjhci-2021-100439).

??? note "A cross-platform informatics system for the Gut Cell Atlas: integrating clinical, anatomical and histological data<br />__Bao, *et al.* (2021) - *Proc. SPIE 11601, Medical Imaging 2021: Imaging Informatics for Healthcare, Research, and Applications*__"
    __Contribution:__ Consulted on scRNA-seq data format/model for integration with imaging and sample metadata.

    __Full Citation:__ Bao S, Chiron S, Tang Y, **Heiser CN**, Southard-Smith AN, Lee HH, Ramirez MA, Huo Y, Washington MK, Scoville EA, Roland JT, Liu Q, Lau KS, Wilson KT, Coburn LA, Landman BA. A cross-platform informatics system for the Gut Cell Atlas: integrating clinical, anatomical and histological data. *Proc. SPIE 11601, Medical Imaging 2021: Imaging Informatics for Healthcare, Research, and Applications*, 1160106. 15 Feb 2021. [doi: 10.1117/12.2581074](https://doi.org/10.1117/12.2581074).

??? note "Coregulator Sin3a promotes postnatal murine β-cell fitness by regulating genes in Ca 2+ homeostasis, cell survival, vesicle biosynthesis, glucose metabolism, and stress response<br />__Yang, *et al.* (2020) - *Diabetes*__"
    __Contribution:__ Analyzed single-cell RNA sequencing data to identify β cell clusters and marker genes using differential gene expression analysis.

    __Full Citation:__ Yang X, Graff SM, **Heiser CN**, Ho K-H, Chen B, Simmons AJ, Southard-Smith AN, David G, Jacobson DA, Kaverina I, Wright CVE, Lau KS, Gu G. 2020. Coregulator Sin3a promotes postnatal murine β-cell fitness by regulating genes in Ca 2+ homeostasis, cell survival, vesicle biosynthesis, glucose metabolism, and stress response. *Diabetes*. p. db190721. [doi: 10.2337/db19-0721](https://doi.org/10.2337/db19-0721).
